lbry/extras/daemon/daemon.py

```diff
@@ -4020,11 +4022,19 @@ def jsonrpc_collection_list(self, resolve_claims=0, account_id=None, wallet_id=N
         wallet = self.wallet_manager.get_wallet_or_default(wallet_id)
         if account_id:
             account = wallet.get_account_or_error(account_id)
-            collections = account.get_collections
+            collections = account.get_collections(resolve=resolve, resolve_claims=resolve_claims)
```

Comment:
I don't think this is correct. Have you tried running this code path? I think it will error because `collections` is expected to be a callable, not the result of a callable.

Reply:
Seems to work making it a partial() - trying with and without account_id, and with various --resolve/--resolve_claims flags.

Reply:
Working on the docs generation - just remembered I had to generate a block between create and list.

Reply:
take a look?